instrument/Facilities.xml

```diff
@@ -929,12 +929,18 @@
   </instrument>
 
 </facility>
+
 <!--  ESS  -->
 <facility name="ESS" FileExtensions=".nxs,.hdf5">
- <timezone>Europe/Copenhagen</timezone>
+  <timezone>Europe/Copenhagen</timezone>
+
   <instrument name="LOKI">
     <technique>Small Angle Scattering</technique>
+    <livedata>
```

Comment:
Do we want to keep these changes?

Reply:
No harm there? Some more sensible default may be good, even if the address needs to be changed (I assume that is possible in the GUI, despite the default?)

Reply:
It can, yes. Changing "Connection" to "[custom]" in the GUI allows this to be changed.